src/main/java/io/debezium/connector/vitess/VitessConnectorConfig.java

```diff
@@ -346,6 +349,15 @@ public static BigIntUnsignedHandlingMode parse(String value, String defaultValue
             .withValidation(Field::isInteger)
             .withDescription("Specify the maximum message size in bytes allowed to be received on the channel.");
 
+    public static final Field GRPC_DEFAULT_LOAD_BALANCING_POLICY = Field.create(VITESS_CONFIG_GROUP_PREFIX + "grpc.default_load_balancing_policy")
+            .withDisplayName("VStream gRPC default load balancing policy")
+            .withType(Type.STRING)
+            .withWidth(Width.MEDIUM)
+            .withDefault(GrpcUtil.DEFAULT_LB_POLICY)
+            .withImportance(ConfigDef.Importance.MEDIUM)
+            .withValidation(VitessConnectorConfig::validateLoadBalancingPolicy)
+            .withDescription("Specify the maximum message size in bytes allowed to be received on the channel.");
```

Comment:
Description is invalid. Also it should probably contain at least two examples of the valid values

Reply:
Nice catch, updated!